lib/src/scaffold.dart

```diff
@@ -572,6 +578,18 @@ class BackdropScaffoldState extends State<BackdropScaffold>
         ),
       ),
     );
+    if (widget.frontLayerBoxShadow == null ||
+        widget.frontLayerBoxShadow!.isEmpty) {
+      return frontPanel;
+    }
+    return _wrapInShadow(frontPanel);
+  }
```

Comment:
should we put `isEmpty` check in constructor asserts - like - `assert(frontLayerBoxShadow == null || frontLayerBoxShadow!.isNotEmpty)`, instead of checking it here?

and just have the following check here:
```suggestion
    if (widget.frontLayerBoxShadow != null) return frontPanel;
    
    return _wrapInShadow(frontPanel);
  }
```

Reply:
@WieFel your views?

Reply:
also in this case not a having a separate `_wrapInShadow` method, but wrapping it here itself.

Reply:
Good idea! LGTM
Haven't tested it, but if it works, it looks perfectly fine!